migration/migration-commands/src/main/java/ddf/migration/commands/MigrateCommand.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.migration.commands;
+
+import ddf.migration.api.DataMigration;
+import ddf.migration.api.ServiceNotFoundException;
+import java.util.Collection;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.Option;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.commands.catalog.SubjectCommands;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Service
+@Command(scope = "migrate-data", name = "run", description = "Runs a specific data migration task")
+public class MigrateCommand extends SubjectCommands {
+  private static final Logger LOGGER = LoggerFactory.getLogger(MigrateCommand.class);
+
+  @Argument(
+    index = 0,
+    name = "serviceId",
+    description = "The service ID of a data migration task",
+    required = true
+  )
+  private String serviceId = null;
+
+  @Option(
```

Comment:
Should we add an option to list all of the data migrations?

Reply:
Added a `-s` option that prints the ID of each service to the console